superset-frontend/src/filters/utils.ts

```diff
@@ -46,8 +46,8 @@ export const getSelectExtraFormData = (
 
 export const getRangeExtraFormData = (
   col: string,
-  lower?: number | null,
-  upper?: number | null,
+  lower?: number | string | null,
+  upper?: number | string | null,
```

Comment:
As the time filter won't be needing this util, we don't need to make changes here.